src/plugins/explore/public/components/visualizations/area/area_vis_options.tsx

```diff
@@ -68,17 +68,12 @@ export const AreaVisStyleControls: React.FC<AreaVisStyleControlsProps> = ({
             />
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <AxesOptions
-              categoryAxes={styleOptions.categoryAxes}
-              valueAxes={styleOptions.valueAxes}
-              onCategoryAxesChange={(categoryAxes) =>
-                updateStyleOption('categoryAxes', categoryAxes)
-              }
-              onValueAxesChange={(valueAxes) => updateStyleOption('valueAxes', valueAxes)}
-              numericalColumns={numericalColumns}
-              categoricalColumns={categoricalColumns}
-              dateColumns={dateColumns}
+            <AllAxesOptions
               axisColumnMappings={axisColumnMappings}
+              standardAxes={styleOptions.standardAxes}
```

Comment:
Seems we are refactor all the axes related config to standardAxes, those legacy visualization schema won't be supported in OSD 4.x I suppose?